DavidZagury/sonic-buildimage
# [NVIDIA] Add support for Service Port in SN5600

#### Why I did it
SN5600 has an additional service interface with a different parameters than other interfaces.

#### How I did it
Added the etp65 interface with the correct parameters.

#### How to verify it
Run platform test on SN5600 platform.
Check the service port can startup correctly.


#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
